src/common/util.ts

```diff
@@ -112,3 +112,11 @@ export const getFirstString = (value: string | string[] | object | undefined): s
 
   return typeof value === "string" ? value : undefined
 }
+
+export function logError(prefix: string, err: any): void {
```

Comment:
What type should `err` be here? `Error | string`?

Reply:
So where this is used err is any as it's logging anything thrown.

Reply:
Nvm, my bad I was wrong, will change to `Error` as that's all that seems necessary for now. Good catch!

Reply:
Though tbh, technically it does accept `any`, there's no reason it has to be `Error` or `string`. Anything else will just be stringified.

Reply:
Let me know if you still want it changed and I can make a follow up PR.